lib/model/QubitInformationObject.php

```diff
@@ -1582,6 +1582,37 @@ public function setActorByName($name, $options)
   }
 
   /**
+   * Import data from a <originalsloc> tag in EAD2002
+   *
```

Comment:
Basically, the locations of originals is a text field and the address fields are unstructured... so there's no way I can see to parse these into AtoM actors. As we don't export these address fields this is just a way to capture information from address tags so we conform to the EAD standard and don't ignore them.